ado_asana_sync/utils/utils.py

```diff
@@ -1,8 +1,9 @@
 """ The util module contains utility functions that are used throughout the application.
 """
+from typing import Any
 
 
-def safe_get(obj: dict | object, *attrs_keys: str) -> any:
+def safe_get(obj: dict | object, *attrs_keys: str) -> Any:
```

Comment:
**Suggestion:** Consider using Union[dict, Any] instead of dict | object for the type hint of 'obj' in the function 'safe_get'. The 'object' type is too broad and doesn't provide much information about what types are expected.
```suggestion
def safe_get(obj: Union[dict, Any], *attrs_keys: str) -> Any:
```